stdlib/public/core/CString.swift

```diff
@@ -91,7 +172,7 @@ extension String {
     guard let str = String._tryFromUTF8(
       UnsafeBufferPointer(start: cString._asUInt8, count: len))
     else { return nil }
-
+    
```

Comment:
?

Reply:
There's still some bits of this that need cleanup :) I forget what used to be there. I'll remove the stray diff though.